### PR TITLE
clean up tablet material entities  and don't display them in create menu

### PR DESCRIPTION
**Pull request by danteruiz**
Since the tablet now uses material entities for highlighting, There are some issues of the material entities persisting after crashes and also being able to see your own tablet material entities in the create app. This PR addresses these two issues.

tickets - https://highfidelity.manuscript.com/f/cases/16517/Material-entities-from-tablet-get-stuck-at-origin

https://highfidelity.manuscript.com/f/cases/16510/VR-users-get-invisible-materialData-objects-attached-to-the-tablet

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11213/HighFidelity-Beta-PR13570-df4d812.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11213/HighFidelity-Beta-PR13570-df4d812.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11213/HighFidelity-Beta-Interface-PR13570-df4d812.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11213/HighFidelity-Beta-PR13570-df4d812.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11213/HighFidelity-Beta-Interface-PR13570-df4d812.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11222/HighFidelity-Beta-PR13570-1cc95fb.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11222/HighFidelity-Beta-PR13570-1cc95fb.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11222/HighFidelity-Beta-Interface-PR13570-1cc95fb.dmg

**Comment by danteruiz** [comment]
Test plan
- launch interface, switch into tablet mode, bring up the tablet.
- switch back to desktop mode, force crash
- launch interface again, (wait until physics is loaded)
- run this script https://hifi-content.s3.amazonaws.com/dante/scripts/findTabletMaterialEntities.js
- open debug log hmd. click on the new tablet app button. 
- the script should print out the number of material avatar entities (no more than 2 should be found)
- If you open create in HMD, you'll be able to see other people material entities but you should not your own. If your in your server less domain you should not see any Tablet-Material-Entities in create.

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11222/HighFidelity-Beta-PR13570-1cc95fb.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11222/HighFidelity-Beta-Interface-PR13570-1cc95fb.exe

**Comment by qlistephens95** [comment]
Currently Testing

**Comment by qlistephens95** [comment]
Currently Testing

**Comment by qlistephens95** [comment]
PASSED @conklin94122. Verified that when script is ran no more than two material avatar entities are found and also verified that when opening create in HMD you cannot see your own material entities.

File: scripts/system/libraries/WebTablet.js [hunk 1]
```diff
@@ -79,6 +81,19 @@ function calcSpawnInfo(hand, landscape) {
     };
 }
 
+
+cleanUpOldMaterialEntities = function() {
+    var avatarEntityData = MyAvatar.getAvatarEntityData();
+    for (var entityID in avatarEntityData) {
+        var entityName = Entities.getEntityProperties(entityID, ["name"]).name;
+
+        if (entityName === TABLET_MATERIAL_ENTITY_NAME && entityID !== HMD.homeButtonHighlightMaterialID &&
```
Comment (sethalves) [inline scripts/system/libraries/WebTablet.js]: Should this maybe also check to make sure the entity is an avatar-entity?

Comment (danteruiz) [reply scripts/system/libraries/WebTablet.js]: I get the entityID from MyAvatar.getAvatarEntityData() which should be avatar entities

Comment (sethalves) [reply scripts/system/libraries/WebTablet.js]: okay, I see.

**Pull request merged**